e2e/scripts/after-create.sql

```diff
@@ -0,0 +1,54 @@
+\set ECHO all
+\set ON_ERROR_STOP 1
+
+-- create a user named bob
```

Comment:
nit: does it make sense to use the name tsdbadmin instead of bob? Makes it more like cloud.

Reply:
i can't believe you don't like "bob"!
jk. yeah, I'll change it